smrf/framework/model_framework.py

```diff
@@ -668,8 +661,27 @@ def initializeOutput(self):
                 self._logger.error(
                     '{} not an output variable'.format(output_variable))
 
+        return variable_dict
+
+    def initializeOutput(self):
```

Comment:
Rename to be consistent: `initialize_output`?

Reply:
There are a few that are still camel case, like `loadTopo` and `loadData`. Perhaps might be better suited for #163 as some are used in many places.